html/pages/syslog.inc.php

```diff
@@ -42,14 +43,20 @@
                         <?php
                         } else {
                             echo '<input type="hidden" name="device" id="device" value="' . $vars['device'] . '">';
+                            $params[] = $vars['device'];
```

Comment:
It makes much more sense to put this under the `$sqlstatement = $sqlstatement . ' WHERE device_id=?';` lines so it's obvious.